### PR TITLE
Feature/dsi 5471 session timeout

### PR DESCRIPTION
**Pull request by malikimran34**

### CODE REVIEW

File: src/app/sharedViews/layout.ejs [hunk 1]
```diff
@@ -224,6 +224,14 @@
             </div>
         </div>
     </div>
+    <div class="session-timeout-overlay">
+        <div class="session-timeout-modal">
+          <h3>Your session will time out in <span id="minutes">5</span> minutes <span id="seconds"></span> seconds </h3>
```
Comment (ipredoi) [inline src/app/sharedViews/layout.ejs:229]: please add class="govuk-heading-m"

File: src/app/sharedViews/layout.ejs [hunk 2]
```diff
@@ -224,6 +224,14 @@
...
+          <p>if you do not respond, we will sign you out. We do this to keep your information safe.</p>
```
Comment (ipredoi) [inline src/app/sharedViews/layout.ejs:230]: please add `class = "govuk-body"`

File: src/app/signOut/views/session-timeout.ejs [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<div class="govuk-width-container">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <br>
+            <h2 class="govuk-heading-xl">
+                Sorry, your session has<br>timed out
+            </h2>
+
+            <h2 class="govuk-heading">Your session automatically ends if you don't do<br>anything for 20 minutes</h2>
```
Comment (ipredoi) [inline src/app/signOut/views/session-timeout.ejs:9]: please update class to `"govuk-heading-m"`

File: src/app/signOut/views/session-timeout.ejs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+            
+            <h2 class="govuk-heading">We do this to keep your information safe.<h2/>
```
Comment (ipredoi) [inline src/app/signOut/views/session-timeout.ejs:11]: please update class to "govuk-heading-m"

File: src/app/signOut/views/session-timeout.ejs [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+            <p class="govuk-body">To use this service again, you need to <a id="signin" href="">sign in</a></p>
```
Comment (ipredoi) [inline src/app/signOut/views/session-timeout.ejs:13]: please add class to the anchor element `class="govuk-link"`

**Review by ipredoi — APPROVED** [review]

**Review by AMcCartneyMTH — APPROVED** [review]

**Pull request merged**